src/PepperDash.Essentials/ControlSystem.cs

```diff
@@ -568,5 +571,62 @@ private bool CheckRoomConfig(DeviceConfig c)
                 return false;
             }
         }
+
+        private static void LoadAssets()
+        {
+            var applicationDirectory = new DirectoryInfo(Global.ApplicationDirectoryPathPrefix);
+            Debug.LogMessage(LogEventLevel.Information, "Searching: {applicationDirectory:l} for embedded assets - {Destination}", applicationDirectory.FullName, Global.FilePathPrefix);
```

Comment:
This can also be simplified to `Debug.LogInformation` (or whatever level is necessary). Skips having an extra using statement at the top.